### DISCUSSION TITLE
Switching to node 18.18.0 using NVM, but Next complain on using different version of node (20.3.1) when starting new project

### DISCUSSION THREAD

**Discussion by GhaziDev** [vercel/next.js#72934] [Help]
### Summary

It is my first time posting on github.

As the title mentioned, I am currently using node 18.18.0 for compatibility with next.js dependencies, i uninstalled every node version possible, and only kept one, yet on starting a new next project using this command:

`npx create-next-app@14.2.17`

I get the following:

![image](https://github.com/user-attachments/assets/22cdb4ac-7f6f-46f3-ae77-f6fd3695a0bf)

![image](https://github.com/user-attachments/assets/a85bbe50-5fb2-4263-a06c-06c3d946b4d8)

i tried uninstall node itself and keep nvm, restarting pc, but nothing worked so far?

**Comment by suprunchuk** [comment]
verify your current Node version:

`node -v`

Then make sure NVM is using Node 18.18.0:

`nvm use 18.18.0`

To set Node 18.18.0 as your default version:

`nvm alias default 18.18.0`

Now, to create a new Next.js project with a specific Node version, you can use the following command which includes the --use-npm flag to ensure npm is used:

`npx create-next-app@14.2.17 --use-npm`

If you still see the version mismatch, you can check all installed Node versions:

`nvm ls`

And remove any unwanted versions:

`nvm uninstall 20.3.1`

To make sure your system is using NVM's Node version and not a globally installed one, check the Node path:

`which node`

These steps should align your Node version with Next.js requirements. Remember to close and reopen your terminal after making these changes for them to take effect.

**Comment by icyJoseph** [comment]
Could you run `node` and then do `process.version`, or perhaps, within the next.js project, in a server component, render `process.version`?

Is `node` in the correct path? Windows is messy like that :/
